### PR TITLE
Add IsFocused/SetFocus APIs to TizenView

### PR DESCRIPTION
**Pull request by JSUYA**
- Add getter and setter of focused state of TizenView
 - Change Unfocus() method to SetFocus().

### CODE REVIEW

**Comment by JSUYA** [comment]
This PR is based on https://github.com/flutter-tizen/engine/pull/352.
The actual created patch is https://github.com/flutter-tizen/engine/pull/354/commits/2f87e1dbac40d39897045722fc218952c4ee9c8e .
I will rebase again when #352 is applied.

**Comment by swift-kim** [comment]
The getter/setter names should be `IsFocused` and `SetFocus`.

**Comment by JSUYA** [comment]
> The getter/setter names should be `IsFocused` and `SetFocus`.

I updated patch

File: shell/platform/tizen/flutter_tizen.cc [hunk 1]
```diff
@@ -274,6 +275,27 @@ void FlutterDesktopViewOnKeyEvent(FlutterDesktopViewRef view,
                               is_down);
 }
 
+void FlutterDesktopViewSetFocus(FlutterDesktopViewRef view_ref, bool focus) {
+  flutter::FlutterTizenView* view = ViewFromHandle(view_ref);
+  auto* tizen_view_base =
+      reinterpret_cast<flutter::TizenViewBase*>(view->tizen_view());
+  if (tizen_view_base->GetType() == flutter::TizenViewType::kView) {
+    auto* tizen_view = reinterpret_cast<flutter::TizenView*>(tizen_view_base);
+    tizen_view->SetFocus(focus);
```
Comment (swift-kim) [inline shell/platform/tizen/flutter_tizen.cc:284]: nit: Too many "view" variables might confuse readers. How about simply

```suggestion
  auto* tizen_view =
      reinterpret_cast<flutter::TizenViewBase*>(view->tizen_view());
  if (tizen_view->GetType() == flutter::TizenViewType::kView) {
    reinterpret_cast<flutter::TizenView*>(tizen_view)->SetFocus(focus);
```

Comment (JSUYA) [reply shell/platform/tizen/flutter_tizen.cc]: done

File: shell/platform/tizen/tizen_view.h [hunk 2]
```diff
@@ -20,7 +20,9 @@ class TizenView : public TizenViewBase {
 
   TizenViewType GetType() override { return TizenViewType::kView; };
 
-  void Unfocus() { focused_ = false; };
+  bool focused() { return focused_; };
+
+  void SetFocus(bool focus) { focused_ = focus; };
```
Comment (swift-kim) [inline shell/platform/tizen/tizen_view.h:25]: focus → focused

**Review by swift-kim — APPROVED** [review]

File: shell/platform/tizen/tizen_view.h (hunk 2 above)
Comment (JSUYA) [reply shell/platform/tizen/tizen_view.h]: thank you for check

**Pull request merged**